client/src/i18n/currency/eur.json

```diff
@@ -0,0 +1,29 @@
+{
+  "CURRENCY_SYM": "€",
+  "DECIMAL_SEP": ".",
```

Comment:
It looks like this should be a comma, right?

Reply:
Fixed